api/cmd/db/migration.go

```diff
@@ -33,6 +33,49 @@ func Migrate(api *app.APIConfig) error {
 		gormigrate.DefaultOptions,
 		[]*gormigrate.Migration{
 			{
+				// Adds Org column and drops Owner from Catalog Table
+				// Adds Unique constraint on (name,org) and
+				// NOT NULL Constraint on other columns
+				ID: "202008071700",
+				Migrate: func(tx *gorm.DB) error {
+					if err := tx.Model(&model.Catalog{}).
+						DropColumn("owner").Error; err != nil {
+						log.Error(err)
+						return err
+					}
+					if err := tx.AutoMigrate(
+						&model.Catalog{}).Error; err != nil {
+						log.Error(err)
+						return err
+					}
+					if err := tx.Model(&model.Catalog{}).
+						AddUniqueIndex("idx_name_org_code", "name", "org").Error; err != nil {
+						log.Error(err)
+						return err
+					}
+
+					catalogQuery := `ALTER TABLE catalogs
```

Comment:
wouldn't automigrate create these?

Reply:
automigrate does not update constraints
 http://gorm.io/docs/migration.html